Composer/packages/tools/language-servers/language-generation/__tests__/helpers/server.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as http from 'http';
+import * as url from 'url';
+import * as net from 'net';
+
+import * as ws from 'ws';
+import * as rpc from 'vscode-ws-jsonrpc';
+import { IConnection, createConnection } from 'vscode-languageserver';
+import express from 'express';
+
+import { LGServer } from '../../src';
+
+function createSocketHandler(webSocket: any): rpc.IWebSocket {
```

Comment:
This function is everywhere, can you refactor it out?

Reply:
It also bother me, it's some util methods help start a LSP Server. 
It's been used at `/server` `/langage-server/demo`, `/language-server/__tests__` .
Surely it should not be built in language-server package, but  do not have a proper place put it.